soh/soh/Enhancements/cosmetics/CustomLogoTitle.cpp

```diff
@@ -205,7 +205,7 @@ void OnZTitleUpdateSkipToFileSelect(void* gameState) {
 
     gSaveContext.seqId = (u8)NA_BGM_DISABLED;
     gSaveContext.natureAmbienceId = 0xFF;
-    gSaveContext.gameMode = GAMEMODE_TITLE_SCREEN;
+    gSaveContext.gameMode = GAMEMODE_FILE_SELECT;
```

Comment:
This was incorrectly the wrong value, but was masked by a different src change below. This should be `GAMEMODE_FILE_SELECT`, which normally would be set by `ovl_en_mag`, but we are skipping the opening sequence so we set it here.